src/ucs/sys/sys.c

```diff
@@ -351,6 +351,24 @@ void ucs_snprintf_zero(char *buf, size_t size, const char *fmt, ...)
     va_end(ap);
 }
 
+void ucs_memunits_to_str(size_t value, char *buf, size_t max)
+{
+    static const char * suffixes[] = {"", "kb", "mb", "gb", "tb"};
```

Comment:
As far as I know, the `kb` should be treated as kilobits (1000bits). If you meant 1024 bytes you have to use `KiB`. Also, the same for `MiB`, `GiB`, `TiB` etc.